crypto/async/arch/async_posix.h

```diff
@@ -25,17 +25,33 @@
 #  define ASYNC_POSIX
 #  define ASYNC_ARCH
 
+#  ifdef __CET__
+/*
+ * When Intel CET is enabled, makecontext will create a different
+ * shadow stack for each context.  async_fibre_swapcontext cannot
+ * use _longjmp.  It must call swapcontext to swap shadow stack as
+ * well as normal stack.
+ */
+#    define USE_SWAPCONTEXT
```

Comment:
Nit: Too many indents here. We indent by one space for each additional pre-processor block - so there should be 3 spaces here.

Reply:
Fixed.